### ISSUE TITLE
Missing semicolons break the code

### ISSUE THREAD

**Issue by SimonHarte** [imakewebthings/waypoints#360]
If you load the files one after another you basically get this:

```javascript
(function() {})()(function() {})()
```
 which results in the following error:

```javascript
TypeError: (intermediate value)(...) is not a function
(function() {
```

Please add semicolons to your code where necessary.

**Comment by imakewebthings** [comment]
This should have already been fixed in #353 and with the release of 3.0.1. Which version are you using?

**Comment by SimonHarte** [comment]
Well ok I downloaded the files from `src` myself instead of using the build file from `lib`.
